osquery/logger/plugins/kafka_producer.cpp

```diff
@@ -0,0 +1,281 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <unistd.h>
+
+#include <atomic>
+#include <future>
+#include <memory>
+#include <mutex>
+
+#include <librdkafka/rdkafka.h>
+
+#include <osquery/config.h>
+#include <osquery/core.h>
+#include <osquery/dispatcher.h>
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+
+namespace osquery {
+
+FLAG(string,
+     logger_kafka_brokers,
+     "localhost",
+     "Bootstrap broker(s) as a comma-separated list of host or host:port "
+     "(default port 9092)");
+
+FLAG(string,
+     logger_kafka_topic,
+     "osquery",
+     "Kafka topic to publish logs under");
+
+FLAG(string,
+     logger_kafka_acks,
+     "all",
+     "The number of acknowledgments the Kafka producer requires the leader to "
```

Comment:
Do you mind making this more terse? Maybe just: "The number of acknowledgments the leader has to receive (0, 1, 'all')"